Common/src/config_structure.cpp

```diff
@@ -1504,6 +1508,10 @@ void CConfig::SetConfig_Options(unsigned short val_iZone, unsigned short val_nZo
   addStringOption("MESH_FILENAME", Mesh_FileName, string("mesh.su2"));
   /*!\brief MESH_OUT_FILENAME \n DESCRIPTION: Mesh output file name. Used when converting, scaling, or deforming a mesh. \n DEFAULT: mesh_out.su2 \ingroup Config*/
   addStringOption("MESH_OUT_FILENAME", Mesh_Out_FileName, string("mesh_out.su2"));
+  /* DESCRIPTION: Determine if the mesh file supports multizone. \n DEFAULT: true (temporarily) */
+  addBoolOption("MULTIZONE_MESH", Multizone_Mesh, true);
```

Comment:
That's exactly the case. The idea is to be able to reuse the single-zone meshes without having to merge them into one single file. Each config reads from its very own MESH_FILENAME. This makes it easier to go from single-zone to multi-zone, and also allows for easier mesh sensitivity studies in the latter. For example, in the FSI case I have split them already (I just merged the PR in TestCases so the tests run correctly here). 
I have maintained the previous format for backwards compatibility (and I guess sometimes it may be better to have all in one mesh file) and it still works well (see the CHT test case).